.github/workflows/pr.yml

```diff
@@ -0,0 +1,40 @@
+name: PR
+
+on:
+  pull_request:
+    branches: [ main ]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+defaults:
+  run:
+    shell: bash -l {0}
+
+jobs:
+
+  Build:
+    name: Build
+    permissions:
+      statuses: write
+      pull-requests: write
+    uses: ./.github/workflows/_build.yml
+
+  Dependabot:
```

Comment:
remove

Reply:
The reason this exists is so that updates to the backpack JS updates are included in the release notes as js updates are excluded by default. This is particularly useful for the svg updates for example

Reply:
_keep_